### DISCUSSION TITLE
跑basic demo中的cli_demo.py推理，多问几次就没有回答了，有人遇到这样的现象吗？

### DISCUSSION THREAD

**Discussion by hsmygcs** [zai-org/ChatGLM3#822] [Q&A] [answered]
![image](https://github.com/THUDM/ChatGLM3/assets/46674492/9aee39fd-2817-4b8a-8435-022b5dc80e2b)

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
你更新最新代码试一下，再看一下显存占用吧，不会有这个问题
